### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by ozerovs**
https://jira.nebenan.de/browse/CORE-5869

### CODE REVIEW

File: src/tooltip/index.es [hunk 1]
```diff
@@ -1,21 +1,58 @@
...
+  const handleClose = useCallback((event) => {
+    event.stopPropagation();
+    setOpen(false);
+  }, [isOpen]);
+
+  const refElement = useRef(null);
+  const refTooltip = useRef(null);
+  const refArrow = useRef(null);
+
+  useEffect(() => {
```
Comment (peterhass) [inline src/tooltip/index.es:33]: I think we should go with `useLayoutEffect` here and provide a proper teardown function

**Comment by peterhass** [comment]
@ozerovs did you test it with dynamic tooltip content? (changing tooltip text / children after the initial render)

**Comment by suprMax** [comment]
I am confused are we going to do it in this repo or the new repo?
If the new - please close/delete branch.
If not the new - why?

**Comment by ozerovs** [comment]
Moved to goodhood

**Pull request closed without merging**